chart_review/commands/frequency.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+
+import rich
+import rich.box
+import rich.table
+import rich.text
+
+from chart_review import cli_utils, console_utils, types
+
+
+def make_subparser(parser: argparse.ArgumentParser) -> None:
+    cli_utils.add_project_args(parser)
+    cli_utils.add_output_args(parser)
+    parser.set_defaults(func=print_frequency)
+
+
+def print_frequency(args: argparse.Namespace) -> None:
+    """
+    Print counts of each text mention.
+    """
+    reader = cli_utils.get_cohort_reader(args)
+
+    frequencies = {}  # annotator -> label -> text -> count
+    all_annotator_frequencies = {}  # label -> text -> count
+    text_labels = {}  # text -> labelset (to flag term confusion)
+    for annotator in reader.annotations.original_text_mentions:
+        annotator_mentions = reader.annotations.original_text_mentions[annotator]
+        for labeled_texts in annotator_mentions.values():
+            for labeled_text in labeled_texts:
+                text = (labeled_text.text or "").strip().casefold()
+                for label in labeled_text.labels:
+                    if label in reader.annotations.labels:
+                        # Count the mention for this annotator
+                        label_to_text = frequencies.setdefault(annotator, {})
+                        text_to_count = label_to_text.setdefault(label, {})
+                        text_to_count[text] = text_to_count.get(text, 0) + 1
+
+                        # Count the mention for our running all-annotators total
+                        all_text_to_count = all_annotator_frequencies.setdefault(label, {})
+                        all_text_to_count[text] = all_text_to_count.get(text, 0) + 1
+
+                        # And finally, add it to our running term-confusion tracker
+                        text_labels.setdefault(text, types.LabelSet()).add(label)
+
+    # Now group up the data into a formatted table
+    table = cli_utils.create_table("Annotator", "Label", "Mention", "Count")
+    has_term_confusion = False  # whether multiple labels are used for the same text
+
+    # Helper method to add all the info for a single annotator to our table
+    def add_annotator_to_table(name, label_to_text: dict) -> None:
+        nonlocal has_term_confusion
```

Comment:
first time i've seen this keyword - i get the approach, i think it makes sense in this use case, but it makes me itchy. 

maybe i just need time to come around on it, like our friend the walrus.